app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -232,48 +218,29 @@ def default_sort
       %w(id desc)
     end
 
+    def index_subscriptions
```

Comment:
This is nitpicky, but is this importing or indexing? Indexing makes me think of our ES setup, import our newer way of handling backend data.